sfge/src/main/java/com/salesforce/rules/AbstractRule.java

```diff
@@ -54,6 +54,11 @@ protected boolean isEnabled() {
         return false;
     }
 
+    protected boolean isExperimental() {
+        // By default, rules are non-experimental.
```

Comment:
Should this be negated? So we are conscious of turning new rules into non-experimental ones?

Reply:
Good point.